eks-worker-al2.json

```diff
@@ -1,42 +1,43 @@
 {
+  "_comment": "All template variables are enumerated here; and most variables have their default values defined in eks-worker-al2-variables.json",
```

Comment:
What's the benefit in duplicating the variables here? Can't `eks-worker-al2-variables.json` be the source of truth?

Reply:
This PR dynamically discovers the variables from the template using `packer inspect`, so they need to be present in the template. ☹️ Because we support overriding them in the Makefile via this mechanism, we have to maintain the list of variables somewhere (either in the Makefile or in the template).

We *could* pull these out with something else like `jq`, but I'd prefer to use the Packer mechanism if it's reasonable.

Reply:
This is the `packer inspect` usage I'm referring to: https://github.com/awslabs/amazon-eks-ami/blob/8da3272cda99241070d5d667e7b75cd3a9e5cf4d/Makefile#L2